### PR TITLE
HDDS-6079. Replace Freon RK with OCKG/OCKV in acceptance tests

### PR DESCRIPTION
**Pull request by adoroszlai**
## What changes were proposed in this pull request?

Replace usage of Freon `randomkeys` with the improved separate subcommands (`ockg` for generating data, `ockv` for verification) in acceptance tests.  This makes it easier to remove `RandomKeyGenerator` in the future, and hopefully increases test stability a bit.

Define Robot keywords for Freon tests for reusability.

https://issues.apache.org/jira/browse/HDDS-6079

## How was this patch tested?

https://github.com/adoroszlai/hadoop-ozone/actions/runs/1560480804

### CODE REVIEW

**Review by fapifta — COMMENTED** [review]
Hi @adoroszlai 

thank you for your work on this patch, I have one concern but it is pretty much outside of the scope for this PR to solve it.

I see that you are removing the run of auditparser tests as it is commented out in the ozone-topology tests. With that I guess we remove the traces of the last place where if was last time running from.
With that there won't be any integration test testing the audit parser. Is this intenional? Shouldn't we run the auditparser tests from somewhere?
I am asking because you also touched the auditparser.robot file, but as that is not running as part of any test suite, I am unsure if it works at all, and if it does not, we should at least note in a JIRA that we should fix it at some point, and re-enable it.
If it works, then why we do not enable it somewhere?

Besides this point, I think the changes are looking good.

**Comment by adoroszlai** [comment]
Thanks @fapifta for the review.

> If it works, then why we do not enable it somewhere?

I don't remember the reason.  Enabled now in the regular `ozone` environment.

**Comment by adoroszlai** [comment]
@fapifta can you please take another look?

File: hadoop-ozone/dist/src/main/smoketest/auditparser/auditparser.robot [hunk 1]
```diff
@@ -48,7 +50,7 @@ Testing audit parser
                        Should Contain       ${result}  ${user}
     ${result} =        Execute              ozone auditparser "${auditworkdir}/audit.db" query "select count(*) from audit where op='CREATE_VOLUME' and RESULT='SUCCESS'"
     ${result} =        Convert To Number     ${result}
-                       Should be true       ${result}>${count}
+                       Should be true       ${result}>=1
```
Comment (smengcl) [inline hadoop-ozone/dist/src/main/smoketest/auditparser/auditparser.robot:53]: This is because we are no longer creating 5 volumes right?

**Review by smengcl — APPROVED** [review]
Thanks @adoroszlai . lgtm

File: hadoop-ozone/dist/src/main/smoketest/auditparser/auditparser.robot (hunk 1 above)
Comment (adoroszlai) [reply hadoop-ozone/dist/src/main/smoketest/auditparser/auditparser.robot:53]: Yes, `ombg` and `ockg` only create one volume.

**Comment by adoroszlai** [comment]
Thanks @fapifta and @smengcl for the reviews.

**Pull request merged**